cpp/ql/src/semmle/code/cpp/exprs/LogicalOperation.qll

```diff
@@ -7,6 +7,9 @@ abstract class UnaryLogicalOperation extends UnaryOperation { }
 
 /**
  * A C/C++ logical not expression.
```

Comment:
```suggestion
 * A C/C++ logical NOT expression.
```

for consistency

Reply:
I'm not sure I agree with this one.  Do we capitalize 'not' anywhere else?

Reply:
The changes right below in the same file change "and" to "AND" and "or" to "OR". The same changes are in `BitwiseOperation.qll`, although bitwise NOT is called _complement_ there.

Reply:
Which would you prefer?  I can make a quick follow-up PR to make this consistent either way.

Reply:
I prefer NOT because it's consistent with AND/OR and because everyone will understand it.